miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -428,12 +439,63 @@ end
 #! tag is the tag to be included in the note.
 #! RECIPIENT is the recipient of the note.
 #! ptr is the pointer to the memory address at which the note is stored.
-export.create_note
+export.create_offchain_note
```

Comment:
I would probably keep one single function for this, but require caller to provide `note_type` argument.